naveed.ini

```diff
@@ -2,7 +2,7 @@
 # * relative paths originate from the application's working directory
 # * changes require restarting the server
 
-host = "localhost"
+host = "0.0.0.0"
```

Comment:
not sure we want this to be the default though?

Reply:
mmm ... ok: your choice. For production localhost doesn't make sense unless this runs on a host that contains a reverse proxy